modules/installation-configuration-parameters.adoc

```diff
@@ -657,6 +657,13 @@ ifdef::ibm-power-vs[]
 endif::ibm-power-vs[]
 |====
 
+ifdef::ibm-power-vs[]
+[NOTE]
+====
+Cloud connections are no longer supported in the `install-config.yaml` while deploying in the `dal10` region, as they have been replaced by the Power Edge Router (PER).
```

Comment:
@alishaIBM This isn't enough to hold up a merge in my opinion, but we generally use a noun after specifying a YAML file. For example: 

```suggestion
Cloud connections are no longer supported in the `install-config.yaml` file while deploying in the `dal10` region, as they have been replaced by the Power Edge Router (PER).
```